Seamless.package/SeamlessSyncRequestContext.class/instance/sendRequest..st

```diff
@@ -5,7 +5,7 @@ sendRequest: aSeamlessRequest
 	
 	self forkProcessingOf: aSeamlessRequest.
 
-	resultWaiter wait.
+	resultWaiter waitTimeoutSeconds: 3.
```

Comment:
I understand the goal. And it's ok. But there are interesting side effects of such timeouts. That's why I don't really address this problem. I will explain it next time.

Now it would be nice to not hardcode timeout value. Try to look how to make it parameter of network and pass it to the context.

Anyway you wrote that image hangs when it waits on this semaphore. Do you know about cmd+dot?